convert-stage-4.sh

```diff
@@ -8,13 +8,14 @@ Mender executables, service and configuration files installer.
 Usage: $0 [options]
 
     Options: [-m|--mender-disk-image | -g|--mender-client | -a|--artifact-name |
-              -d|--device-type | -p|--demo-host-ip | -u| --server-url |
+              -d|--device-type | -e|--demo | -p|--demo-host-ip | -u| --server-url |
```

Comment:
'-e' option is reserved in the main 'mender-convert' script for 'storage-total-size-mb' - so maybe it is worth to not mix that